.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/node:8
+      - image: circleci/node:10
```

Comment:
I was wondering why this was included, but I'm guessing it's because `URL` wasn't in Node v8

Reply:
correct